sci-biology/infernal/infernal-1.0.2-r1.ebuild

```diff
@@ -28,7 +28,7 @@ src_prepare() {
 
 src_configure() {
 	econf \
-		--prefix="${D}/usr" \
+		--prefix="${ED}/usr" \
```

Comment:
this is the wrong fix. Please port to EAPI 6 and drop the `--prefix` specifier (as it's implicitly set correctly)